docs/api-communication-spec.md

```diff
@@ -0,0 +1,686 @@
+| `CAPTURE_STOPPED`       | Confirmation that the capture has stopped gracefully. All of the targeted agents have stopped.                                                                    |
+| `LIMIT_REACHED`         | Some limit has been reached, e.g. number of concurrent requests, time, bytes, etc.; Message details identifies, which limit has been reached.                     |
+| `UNAUTHORIZED`          | The token sent by the client is rejected (e.g. invalid, timed out, etc.). Detail for the rejection in the message.                                                |                                                                                                     |
+| `MTLS_ERROR`        | An error happened while attempting mTLS communication with PCAP components, independent of the client.         |
+| `NO_CAPTURE_RUNNING`    | A Stop Capture Request is received but no capture is running.                                                                                                     |
+
+### Status
+
+| Parameter | Type          | Required? | Description                                                                                                                             |
+|-----------|---------------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| `health`  | `enum Health` | yes       | The type of message sent. This allows for logic based on the message type                                                               |
+| `version` | `string`      | yes       | Version number of the component. Can be used to ensure communication with compatible versions, and cut-off of unsupported old versions. |
+| `status`  | `string`      | yes       | A human readable status message.                                                                                                        |
+| `cf`      | `boolean`     | no        | Supports CF requests (only for pcap-api)                                                                                                |                                                                                               | 
+| `bosh`    | `boolean`     | no        | Supports BOSH requests (only for pcap-api)                                                                                              |                                                                                               | 
+
+#### Enum `Health`
+
+| Value       | Description                                                                                                                                  |
+|-------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| `OK`        | Everything is nominal.                                                                                                                       |
```

Comment:
If we use `UNHEALTHY` this should be `HEALTHY`

Reply:
agreed.